src/mutable_data.rs

```diff
@@ -50,6 +51,14 @@ impl Value {
     pub fn new(data: Vec<u8>, version: u64) -> Self {
         Value { data, version }
     }
+
+    pub fn data(&self) -> Vec<u8> {
+        self.data.clone()
```

Comment:
Making `data` pub in struct would be preferable, but we'll make this change in one of the next PRs.